spec/factories/user.rb

```diff
@@ -17,7 +18,7 @@
       if e.miq_groups.blank? && (e.role || e.features)
         u.miq_groups = [
           (e.role && MiqGroup.find_by_description("EvmGroup-#{e.role}")) ||
-          FactoryGirl.create(:miq_group, :features => e.features, :role => e.role)
+          FactoryGirl.create(:miq_group, :features => e.features, :role => e.role, :tenant => e.tenant)
```

Comment:
Doesn't this imply that if you specify a user with a role or features, but *dont* specify a tenant, that the resulting group will be tenantless (you're specifying `nil` regardless)?